src/models/RevAiApiClientConfig.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-shadow */
+/**
+ * Configuration object to initialize RevAiApiClient with
+ * @param token Access token used to validate API requests, note access token should match the deployment
+ *    associated with the url in @param baseUrl
+ * @param version version of the API to be used
+ * @param baseUrl base url of the API to be used. Note base urls are different for Rev AI deployments in
+ *     different locations. Base urls are also different for asynchronous and streaming APIs.
+ * @param serviceApi Type of api service
+ */
+export interface RevAiApiClientConfig {
+    token?: string;
+    version?: string;
+    baseUrl?: string;
```

Comment:
I think this should be region as I stated above. Additionally, this would allow us to handle url switching for async and streaming internally, rather than users needing to configure different url.